### PR TITLE
loop waiting for containerd server to respond to client connection 

### PR DESCRIPTION
**Pull request by mikebrow**
Adds a loop that increases in delay five times while attempting to wait for the containerd server to come up enough that it can respond to a version request from the client.

Signed-off-by: Mike Brown <brownwm@us.ibm.com>

### CODE REVIEW

File: hack/test-cri.sh [hunk 1]
```diff
@@ -26,9 +26,9 @@ SKIP=${SKIP:-"Streaming|RunAsUser|host port"}
 REPORT_DIR=${REPORT_DIR:-"/tmp"}
 
 if [[ -z "${GOPATH}" ]]; then
-  echo "GOPATH is not set"	
+  echo "GOPATH is not set"
```
Comment (Random-Liu) [inline hack/test-cri.sh]: Thanks for the cleanup!

File: hack/test-cri.sh [hunk 2]
```diff
@@ -62,9 +62,17 @@ fi
 sudo pkill containerd
 sudo containerd -l debug &> ${REPORT_DIR}/containerd.log &
 
+MAX_ATTEMPTS=5
+attempt_num=1
+until sudo ctr version | grep -m 1 "Server:" || (( attempt_num == MAX_ATTEMPTS ))
```
Comment (Random-Liu) [inline hack/test-cri.sh]: Good idea to use `ctr`. No need to grep, just:
```shell
until sudo ctr version &> /dev/null || (( attemp_num == MAX_ATTEMPTS ))
do
  ...
done
```

Comment (mikebrow) [inline hack/test-cri.sh]: kk.. will check that out in the morning :-)

**Comment by mikebrow** [comment]
rebased.. removed sleep 1 from master, updated to address above comment from @Random-Liu...

**Comment by Random-Liu** [comment]
LGTM

**Pull request merged**